gen/dcos-config.yaml

```diff
@@ -2785,6 +2785,7 @@ package:
           # are further forwarded into elasticsearch, where fields with leading
           # underscore are reserved for internal use.
           Strip_Underscores On
+          Read_From_Tail    On
```

Comment:
Why this change instead of setting `DB` to some value in order to configure a persistent cursor?

Reply:
We should enable the DB (this will require setting some build flags, as [FLB_SQLDB is not enabled by default](https://github.com/fluent/fluent-bit/blob/master/CMakeLists.txt#L62)), but we would still need the `Read_From_Tail` flag - when a user upgrades to 1.13, they will have logs on disk which are unread, and it's this read that caused issues in soak.

Reply:
Understood. It sounds like we should have a JIRA (if we don't already) to track the tasks associated with enabling the `DB` flag here. Can you link such to this comment thread?

Reply:
https://jira.mesosphere.com/browse/DCOS-51855